### PR TITLE
Reformatting using scalariform

### PR DESCRIPTION
**Pull request by mecampbellsoup**
Another no-op PR here.

Now when running `compile` or `test:compile` [scalariform](https://github.com/scala-ide/scalariform) will automatically
reformat the source code files.

See:

* https://github.com/scala-ide/scalariform
* https://github.com/sbt/sbt-scalariform
* [Default formatter preferences](https://github.com/scala-ide/scalariform/blob/master/formatterPreferences.properties)

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -9,10 +9,12 @@ object BitcoinSCoreBuild extends Build {
   val slf4jV = "1.7.5"
   val logbackV = "1.0.13"
   val scalaTestV = "2.2.0"
+  //val scalariformV = "1.8.2"
```
Comment (Christewart) [inline project/Build.scala]: Can we get rid of these comments?

**Comment by Christewart** [comment]
I ran this locally and everything passed. Can you address the nits and then I'll merge.

**Comment by Christewart** [comment]
You will have to rebase onto master now.

**Comment by mecampbellsoup** [comment]
@Christewart done :)

**Pull request merged**